lib/utils.js

```diff
@@ -64,12 +64,21 @@ var utils = module.exports = {
         simpleGit(cwd).revparse([ '--abbrev-ref', 'HEAD' ], callback)
       })
         .then(function (branch) {
-          return branch.toString().trim()
+          if (branch) {
+            return branch.toString().trim()
+          }
+          return branch
```

Comment:
how was this not returning anything for you? it's curious to me that you need this. outside of an error, this should (I believe) almost always return at least a `Buffer`